docker-compose.dev.yml

```diff
@@ -7,14 +7,31 @@ services:
     # Flask development server with live debugger and hot reloading:
     command: "flask run --host 0.0.0.0"
     build: "."
+    # To make volume mounting (especially for new volume mounts like the DB)
+    # easier, override the micromamba non-root user:
+    user: "root"
+    volumes:
+      - "./usaon_vta_survey:/usaon-vta-survey/usaon_vta_survey:ro"
+      - "./tasks:/usaon-vta-survey/tasks:ro"
+      - "./_db:/db:rw"
     ports:
       - "5000:5000"
     environment:
+      # Disable login:
+      - "USAON_VTA_LOGIN_DISABLED=true"
+      # Use a local file DB instead of a remote one:
+      - "USAON_VTA_DB_SQLITE=true"
+      # Enable the in-browser debugger:
       - "FLASK_DEBUG=True"
+      # Disable error handlers and allow exceptions to propagate:
+      - "FLASK_TESTING=True"
```

Comment:
I believe this can be removed. I think it has no effect. I thought it would, following the pattern of `FLASK_DEBUG`, result in `app.config['TESTING']` being set to `True`, but I think it doesn't.